blog/_posts/2017-05-19-introducing-the-new-goodtables-library-and-goodtablesio.md

```diff
@@ -2,102 +2,105 @@
-This kind of continuous feedback allows data publishers to release better, higher quality data and helps ensure that this quality is maintained over time, even if different people publish the data. 
+goodtables v1 also works with our proposed [data quality specification standard](https://github.com/frictionlessdata/data-quality-spec), which has [an extensive list](https://github.com/frictionlessdata/goodtables-py/blob/master/goodtables/spec.json) of standard tabular data errors.  Other enhancements from goodtables v0.7 include:
 
-Using [this dataset on Github](http://goodtables.io/github/frictionlessdata/example-goodtables.io), here’s sample output from data validation run on goodtables.io:
+1. Breaking out [tabulator](http://github.com/frictionlessdata/tabulator) into its own library.  As part of the Frictionless Data framework, **_tabulator_** is a python library that has been developed to provide a consistent interface for stream reading and writing tabular data that is in whatever format, be it CSV, XML, etc. The library is installable via pip: `pip install tabulator`.
+2. Close to 100% support for  [Table Schema](http://specs.frictionlessdata.io/table-schema/) due to lots of work on the underlying [Python library](https://github.com/frictionlessdata/jsontableschema-py).  The Table Schema Python library allows users to validate dataset schema and, given headers and data, infer a schema as a python dictionary based on its initial values.
+3. Better CSV parsing, better HTML detection, and less false positives
+
+## goodtables.io
+
+![goodtablesio](/img/posts/goodtablesio.jpg)
 
-![illustrating data validation on goodtables.io](https://github.com/okfn/okfn.github.com/blob/master/img/posts/gtio%20data%20validation%20.png)
+Moving forward, at Open Knowledge International we want to streamline the process of data validation and ensure seamless integration is possible in different publishing workflows. To do so, [we are launching a continuous data validation hosted service](https://discuss.okfn.org/t/launching-goodtables-io-tell-us-what-you-think/5165) that builds on top of all this suite of Frictionless Data libraries.  [goodtables.io](http://goodtables.io) will provide support for different backends. At this time, users can use it to check any datasets hosted on GitHub and Amazon S3 buckets, automatically running validation against data files every time they are updated, and providing a user friendly report of any issues found.
+
+Try it here: [goodtables.io](http://goodtables.io)
+
+This kind of continuous feedback allows data publishers to release better, higher quality data and helps ensure that this quality is maintained over time, even if different people publish the data.
+
+Using [this dataset on Github](http://goodtables.io/github/frictionlessdata/example-goodtables.io), here’s sample output from data validation run on goodtables.io:
```

Comment:
If you want a more exciting example you can use this one:

http://goodtables.io/github/amercader/car-fuel-and-emissions

repo: https://github.com/amercader/car-fuel-and-emissions